superset-frontend/src/profile/components/App.tsx

```diff
@@ -32,6 +31,14 @@ interface AppProps {
   user: UserWithPermissionsAndRoles;
 }
 
+const StyledPanel = styled.div`
+  padding: 10px;
+  .ant-tabs-content-holder {
```

Comment:
If the above line of padding comes out, then I wouldn't use the styled component as implemented here. Rather than making a styled component which wraps all the `Tabs.TabPane` components, and affecting them with this `.ant-tabs-content-holder` class selector, I would make a `StyledTabPane` with just the two CSS properties, and use that in the JSX markup directly instead of `TabPane`

Reply:
yep!